lib/node_modules/@stdlib/stats/base/dists/planck/pmf/README.md

```diff
@@ -0,0 +1,144 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Probability Mass Function
+
+> Planck distribution [probability mass function][pmf] (PMF).
+
+<section class="intro">
+
+The [probability mass function][pmf] (PMF) for a planck random variable is defined as
+
+<!-- <equation class="equation" label="eq:planck_pmf" align="center" raw="\Pr(X = x, \lambda) = \begin{cases}(1 - e^{-\lambda})e^{-\lambda x} & \text{for } x = 0, 1, 2, \ldots \\ 0 & \text{otherwise} \end{cases}" alt="Probability mass function (PMF) for a Planck distribution."> -->
+
+```math
+\Pr(X = x, \lambda) = \begin{cases}(1 - e^{-\lambda})e^{-\lambda x} & \text{for } x = 0, 1, 2, \ldots \\ 0 & \text{otherwise} \end{cases}
+```
+
+<!-- </equation> -->
+
+where `lambda > 0` is the shape parameter. The random variable `X` denotes the count of events in a quantized system.
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var pmf = require( '@stdlib/stats/base/dists/planck/pmf' );
+```
+
+#### pmf( x, p )
+
+Evaluates the [probability mass function][pmf] (PMF) of a planck distribution with shape parameter `lambda`.
+
+```javascript
+var y = pmf( 4.0, 0.3 );
+// returns ~0.0781
+
+y = pmf( 2.0, 1.7 );
+// returns ~0.0273
+
+y = pmf( -1.0, 2.5 );
+// returns 0.0
+```
+
+If provided `NaN` as any argument, the function returns `NaN`.
+
+```javascript
+var y = pmf( NaN, 0.0 );
+// returns NaN
+
+y = pmf( 0.0, NaN );
+// returns NaN
+```
+
+If provided a shape parameter `lambda` which is negative number, the function returns `NaN`.
+
+```javascript
+var y = pmf( 2.0, -1.0 );
+// returns NaN
+```
+
+#### pmf.factory( p )
+
+Returns a function for evaluating the [probability mass function][pmf] (PMF) of a planck distribution with shape parameter `lambda`.
+
+```javascript
+var mypmf = pmf.factory( 0.5 );
+var y = mypmf( 3.0 );
+// returns ~0.0878
+
+y = mypmf( 1.0 );
+// returns ~0.2387
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var randu = require( '@stdlib/random/base/randu' );
+var round = require( '@stdlib/math/base/special/round' );
```

Comment:
Let's go ahead and generate an array of random numbers using `@stdlib/random/array/discrete-uniform`. This ensures consistency with some of the recent work we've been doing when updating various dists packages.